### PR TITLE
fusuma: 1.3.0 -> 2.4.1, fusumaPlugins init

### PR DESCRIPTION
**Pull request by LazyGeniusMan**
###### Description of changes

maintainers: init LazyGeniusMan
fusuma: 1.3.0 -> 2.4.1
fusumaPlugins.appmatcher: init at 0.4.0
fusumaPlugins.keypress: init at 0.5.0
fusumaPlugins.sendkey: init at 0.6.4
fusumaPlugins.tap: init at 0.4.2 
fusumaPlugins.wmctrl: init at 1.1.0 

###### Things done

- Built on platform(s)
  - [x] x86_64-linux (https://github.com/LazyGeniusMan/nixpkgs-test/actions/runs/3032150781)
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/inputmethods/fusuma/plugins.nix [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  wmctrl = bundlerApp {
+    pname = "fusuma-plugin-wmctrl";
+    gemdir = ./fusuma-plugin-wmctrl;
+    exes = [ "fusuma-wmctrl" ];
+
+    nativeBuildInputs = [ makeWrapper ];
+
+    postBuild = ''
+    wrapProgram "$out/bin/fusuma-wmctrl" \
+      --prefix PATH : ${lib.makeBinPath [ libinput fusuma ]}
```
Comment (SuperSandro2000) [inline pkgs/tools/inputmethods/fusuma/plugins.nix:90]:
```suggestion
      wrapProgram "$out/bin/fusuma-wmctrl" \
        --prefix PATH : ${lib.makeBinPath [ libinput fusuma ]}
```

File: pkgs/tools/inputmethods/fusuma/plugins.nix [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+  sendkey = bundlerApp {
+    pname = "fusuma-plugin-sendkey";
+    gemdir = ./fusuma-plugin-sendkey;
+    exes = [ "fusuma-sendkey" ];
+
+    nativeBuildInputs = [ makeWrapper ];
+
+    postBuild = ''
+    wrapProgram "$out/bin/fusuma-sendkey" \
+      --prefix PATH : ${lib.makeBinPath [ libinput fusuma ]}
```
Comment (SuperSandro2000) [inline pkgs/tools/inputmethods/fusuma/plugins.nix:51]:
```suggestion
      wrapProgram "$out/bin/fusuma-sendkey" \
        --prefix PATH : ${lib.makeBinPath [ libinput fusuma ]}
```

File: pkgs/tools/inputmethods/fusuma/plugins.nix [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    pname = "fusuma-plugin-appmatcher";
+    gemdir = ./fusuma-plugin-appmatcher;
+    exes = [ "fusuma-appmatcher" ];
+
+    nativeBuildInputs = [ makeWrapper ];
+
+    postBuild = ''
+    wrapProgram "$out/bin/fusuma-appmatcher" \
+      --prefix PATH : ${lib.makeBinPath [ libinput fusuma ]}
+      '';
```
Comment (SuperSandro2000) [inline pkgs/tools/inputmethods/fusuma/plugins.nix:13]:
```suggestion
      wrapProgram "$out/bin/fusuma-appmatcher" \
        --prefix PATH : ${lib.makeBinPath [ libinput fusuma ]}
    '';
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by LazyGeniusMan** [comment]
oops, I misclicked and it's removed the other reviewer (previous maintainer)

**Review by nicknovitski — APPROVED** [review]

**Pull request closed without merging**